.gitignore

```diff
@@ -1,6 +1,5 @@
 __pycache__/
 *.pyc
 *.log
-*DS*Store*
-
+.DS_Store
```

Comment:
.DS_Store has been already committed to the repository already. This somehow makes .gitignore to fail with any pattern. I don't understand why it fails. I have had files stored in repository which are also in ignored if modified.

In this format it also matches src/,DS_Store and test/.DS_Store [1]. 

[1] https://git-scm.com/docs/gitignore#_examples